src/core/core.ts

```diff
@@ -19,7 +19,8 @@ class Core {
     public static readonly version = {
         version: process.env.VERSION,
         revision: process.env.COMMIT_HASH,
-        branch: process.env.COMMIT_BRANCH
+        branch: process.env.COMMIT_BRANCH,
+        buildId: process.env.ADYEN_BUILD_ID
```

Comment:
Shall we also add the name of the library here so it can be easily found?

Reply:
Good you mentioned @pabloai! That's been included in the `buildId` (e.g. `@adyen/adyen-web-f59434c6-1af3-...`)